pages/learn/develop/hardware/i2c-and-spi.md

```diff
@@ -348,6 +350,16 @@ After the custom device tree has been validated, it can be included in newer bal
 
 Please note that if the changes for your carrier board expand past kernel device-trees, or require modifications to board configuration files like pin multiplexing configuration files or any other device-trees or files used by firmware, these may not be provided by the existing cloud images. During provisioning the resulting configuration changes are stored in the QSPI or in the hardware defined boot partitions, and thus will be replaced with the default values when updating the Host Operating System. Please [contact us](https://www.balena.io/contact-sales) if you would like to use a Jetson carrier board which may not be fully compatible with its' corresponding devkit, or with any of our cloud images for your Jetson module.
 
+### Configurable fan profiles
+
+Jetson Orin Devices running balenaOS revisions newer than <TODO> and supervisor versions greater than <TODO> offer support for configurable fan profiles. You can switch between the two options provided by Jetpack: "quiet" and "cool", by navigating to the "Device Configuration" page on the balenaCloud dashboard and selecting the desired option. The change is performed at runtime, and does not trigger a device reboot. Preloading the [fan profile configuration](/reference/OS/configuration/#fanprofile) before provisioning your device can also be achieved by editing [config.json](/reference/OS/configuration/#about-configjson)
+
+### Configurable power modes
+
+The same balenaOS versions for Jetson Orin which support configurable fan profiles also offer the possibility for selecting the desired power mode. You can choose between "low", "mid" and "high" or specify the power mode index directly by navigating to the "Device Configuration" page on the balenaCloud dashboard. The available power modes indexes for your device type are visible in the host OS in `/etc/nvpmodel.conf`. Similar to the fan profile, preloading the desired power mode configuration can be achieved by editing [config.json](/reference/OS/configuration/#about-configjson) and specifying the desired [power mode](/reference/OS/configuration/#powermode) before provisioning the device.
+
+NOTE: In order for the selected configuration to be applied, the device will automatically reboot after you have chosen a different power mode.
```

Comment:
Better to add this to the section above only and not as a note.

Reply:
done